lib/shopify.ex

```diff
@@ -82,5 +86,7 @@ defmodule Shopify do
       iex> with %Shopify.Session{} <- Shopify.session, do: :passed
       :passed
   """
-  def session, do: Shopify.Session.new()
+  def session do
```

Comment:
Is there a reason you prefer this version over the one line version? Personal preferences?

Reply:
More of a personal preference. I generally like the consistency of the multi-line function.